frontend/compositions/dashboardHeader/header.module.css

```diff
@@ -0,0 +1,156 @@
+/* default - mobile styles */
+.wrapper {
+  width: 100%;
+  height: 6.94rem;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.backgroundBanner {
+  width: 100%;
+  position: absolute;
+  z-index: -10000;
+  background-image: url("./assets/BannerMobile.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 4.74rem;
+  filter: drop-shadow(0rem 0.25rem 0.25rem rgba(0, 0, 0, 0.25));
+}
+
+.leftHeader {
+  padding-left: 0.82rem;
+  margin-top: 2.15rem;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.titleContainer {
+  margin-top: 0.22rem;
+  padding-left: 0.56rem;
+  width: 8.81rem;
+  height: 3.88rem;
+}
+
+.titleContainer h2 {
+  color: var(--darkBlue);
+  font-weight: 700;
+  font-style: normal;
+  font-size: 1.75rem;
+  line-height: 2.12rem;
+  letter-spacing: -0.02rem;
+}
+
+.titleContainer p {
+  width: 6.9rem;
+  height: 1.63rem;
+  color: var(--black);
+  font-weight: normal;
+  font-size: 0.63rem;
+  line-height: 0.75rem;
+  letter-spacing: -0.02rem;
+}
+
+.rightHeader {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.nav {
+  margin: 1.69rem 1.69rem 1.8rem;
+}
+
+.button {
```

Comment:
There's a new global class for `primaryButton` you could use here if you like! You might have to specify sizing still but might DRY up the rest!

Reply:
Updated to use the global class